src/App.js

```diff
@@ -1,6 +1,8 @@
 import React from 'react';
-import './assets/scss/App.scss';
 import { Routes, Route } from 'react-router-dom';
+
```

Comment:
- [ ]  Your app runs with errors. I'm not able to see the home page.  After running the `npm install` then `npm start`.  Actually, I'm getting this error message 👇🏻 Please ensure that your app works as expected and runs without errors. 😉 👍🏻 

- Browser: 
![conso](https://user-images.githubusercontent.com/79058364/179125868-8faa97ea-9db5-4d85-ab08-8c81a844e893.png)

Reply:
Hello, that sounds that a problem on your end, I've tested the app in a couple of devices and it works properly.